packages/ts-client/src/webrtc/webRTCEndpoint.ts

```diff
@@ -326,14 +326,13 @@ export class WebRTCEndpoint<EndpointMetadata = any, TrackMetadata = any> extends
     this.remote.updateMLineIds(data.midToTrackId);
     this.local.updateMLineIds(data.midToTrackId);
 
-    Object.values(data.midToTrackId)
+    Object.keys(data.midToTrackId)
```

Comment:
Was this change required because of other changes in this PR, or is it separate bug?